selenium/Dockerfile

```diff
@@ -52,4 +68,4 @@ ENV DISPLAY :20.0
 CMD /usr/bin/supervisord -c /opt/supervisord.conf & \
     cd /home/user && sleep 3 && \
     ./chromedriver --port=4444 --whitelisted-ips='' & \
-    sleep 365d
\ No newline at end of file
+    sleep 365d
```

Comment:
replace `sleep 365d` with `tail -f /dev/null`
add new line at the end of the file